src/stream/src/executor/dml.rs

```diff
@@ -135,6 +142,14 @@ impl DmlExecutor {
                                 _ => {}
                             }
                         }
+
+                        // Flush the remaining batch group
+                        if !batch_group.is_empty() {
+                            let vec = mem::take(&mut batch_group);
+                            let concat_chunk = StreamChunk::concat(vec);
```

Comment:
`concat` seems to be a method for test purposes.

Reply:
Yes, can we just move this method to `impl StreamChunk`?

Reply:
It uses `rechunk()` internally, while it's rechunk's comment writes:

```
    /// Currently, `rechunk` would ignore visibility map. May or may not support it later depending
    /// on the demand
```

🥵

Reply:
🥵

Reply:
Perhaps we need to write a new one.

Reply:
Though it doesn't matter here, chunks from txn_buffer should be all visible. Let's me use stream chunk builder instead.

Reply:
> It uses `rechunk()` internally, while `rechunk`'s comment writes:

Also it does not handle `Op::UpdateDelete` correctly.